### ISSUE TITLE
createOlStyle method in addition to createOlStyleFunction

### ISSUE THREAD

**Issue by Feofilakt** [NieuwlandGeo/SLDReader#147]
Hello! Is there some way to retrieve from SLD OL Style instead of StyleFunction, like "createOlStyle" method? In case we know that sld doesn't depend on resolution or feature attributes.

**Comment by ajkopinga** [comment]
There currently is no equivalent of `createOlStyle` that does not require a feature. I think a 'static' createOlStyle function could be a useful addition.

But even if a rule or symbolizer does not depend on feature properties, information about the feature geometry is still needed to convert the Point, Line or Polygon symbolizers to the correct OL style. You'd have to provide information about the geometry type of features you'll be applying the style(s) to.

Would something like this be useful for you?
```javascript
// Returns array of line styles corresponding to the LineSymbolizers in the style rule.
createOlStyle(featureTypeStyle.rules[0], 'Line'); // or Point or Polygon.
```

**Comment by Feofilakt** [comment]
Tell me if I understand correctly: in SLD you basically set symbolizer for concrete geometry types, i.e. PointSymbolizer for points, LineSymbolizer for (multy)lines and PolygonSymbolizer for (multy)polygons. And OL has not Symbolizers concept, it has universal style primitives, for example stroke that can be used for both lines and polygons. So SldReader split SLD into some  "switch set" and applies rules from symbolizers based on feature geometry type like mentioned above?

Yes, in this case providing geometry type makes sense. It would be great to have such functionality.

**Comment by ajkopinga** [comment]
I've released version 0.2.19 that contains this new `createOlStyle` method.
